### PR TITLE
workaround for occasional and puzzling Empty exceptions being raised …

### PR DESCRIPTION
**Pull request by amanningeso**
Another one I can't reliably reproduce. I saw this while abusing the mirroring system again with rapid restarts in combination with dropping packets. Another one where I think a failure in ALMA mirroring provoked IPC issues.

### CODE REVIEW

**Comment by rtobar** [comment]
In principle this shouldn't happen, but it does because the thread on the main process that handles IPC with the janitor process starts *after* the janitor process is launched, opening the window for a race condition where a message sent by the janitor process is now handled by the main process in time.

https://github.com/ICRAR/ngas/blob/fdd6726c08d5778503c4cd01fdd2e168e97e07e2/src/ngamsServer/ngamsServer/ngamsServer.py#L1065-L1074

In other words: if I add a `time.sleep` in line 1074 above I can reliably reproduce this problem.

@amanningeso if you agree, I'll drop this PR (it also conflicts with #76 anyway) in favor of correcting the start sequence of the janitor communication thread and janitor process.

**Comment by amanningeso** [comment]
I agree.

**Comment by rtobar** [comment]
Closing in favor of 3359cc0
